Description: Don't even think about using distribute_setup to download things during build
Author: Stefano Rivera <stefanor@debian.org>
Forwarded: not-needed
Last-Update: 2012-02-03

--- a/setup.py
+++ b/setup.py
@@ -7,11 +7,7 @@
 test_dirs = ['functional_tests', 'unit_tests', os.path.join('doc','doc_tests'), 'nose']
 
 if sys.version_info >= (3,):
-    try:
-        import setuptools
-    except ImportError:
-        from distribute_setup import use_setuptools
-        use_setuptools()
+    import setuptools
 
     extra = {'use_2to3': True,
              'test_dirs': test_dirs,
